File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    nephew = ["aya", "ye", "woo", "ma"]
    result = 0
    
    for i in range(len(babbling)):
        for j in range(len(nephew)):
            if nephew[j] + nephew[j] in babbling[i]:
                break;
            babbling[i] = babbling[i].replace(nephew[j], ' ')
        if babbling[i].replace(' ', '') == '':
            result += 1
    return result

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.js
function solution(babbling) {
    const nephew = ["aya", "ye", "woo", "ma"];
    let result = 0;
    
    for (let i in babbling) {
        for (let j in nephew) {
            if (babbling[i].includes(nephew[j] + nephew[j])) break;
            babbling[i] = babbling[i].replaceAll(nephew[j], ' ');
        }
        
        if (babbling[i].replaceAll(' ', '') === '') {
            result++;
        }
    }
    return result;
}